packages/sdk/src/__tests__/integration/modules/moduleFromEndpoints.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { initSDK, buildModule } from "../../../index";
+import { moduleFromEndpoints } from "../../../modules/moduleFromEndpoints";
+
+type Endpoints = {
+  getProduct: (params: { id: string }) => Promise<any>;
+  getProducts: (params: { limit: number }) => Promise<any>;
+};
+
+describe("moduleFromEndpoints", () => {
+  it("should be able to be used as standard SDK module", async () => {
+    const sdkConfig = {
+      commerce: buildModule(moduleFromEndpoints),
+    };
+
+    const sdk = initSDK(sdkConfig);
+
+    expect(sdk.commerce).toBeDefined();
+  });
+
+  it("should use generic types to define the endpoints", async () => {
+    const sdkConfig = {
+      commerce: buildModule(moduleFromEndpoints<Endpoints>),
+    };
+
+    const sdk = initSDK(sdkConfig);
+
+    expect(sdk.commerce.getProduct).toBeDefined();
+    expect(sdk.commerce.getProducts).toBeDefined();
```

Comment:
I miss type tests, to check if the correct type is inferred, but afaik `jest` doesn't have such feature. Maybe could we at least check if it is function

```suggestion
    expect(sdk.commerce.getProduct).toBeInstanceOf(Function);
    expect(sdk.commerce.getProducts).toBeInstanceOf(Function);
```